azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/common/audiodevicelist/AudioDeviceListViewModel.kt

```diff
@@ -14,6 +15,11 @@ internal class AudioDeviceListViewModel(
     private val dispatch: (Action) -> Unit,
 ) {
 
+    val receiverTypeTitle get() = when (audioStateFlow.value.isHeadphonePlugged) {
```

Comment:
Adam, it will be great if we can avoid view classes in view model.

Reply:
Is string resource id considered a view class? I can push down into the view.